import-service/src/functions/importProductsFile/handler.ts

```diff
@@ -0,0 +1,52 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import { PutObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import {
+  getSignedUrl,
+} from "@aws-sdk/s3-request-presigner";
+
+import schema from './schema';
+
+const importProductsFile: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+  try{
+    if(!event.queryStringParameters.name) {
+      return formatJSONResponse({
+        statusCode: 500,
+        body: { error: 'Name is not provided' },
+        headers: {
+          'Access-Control-Allow-Origin': '*',
+        },
+      });
+    }
+  
+    const createPresignedUrlWithClient = ({ region, bucket, key }) => {
+      const client = new S3Client({ region });
+      const command = new PutObjectCommand({ Bucket: bucket, Key: key });
+      return getSignedUrl(client, command, { expiresIn: 3600 });
```

Comment:
I've never faced SignedUrls before, and i'm not quite sure about the secureness of this approach.

Maybe you know, is it at least secured to provide `put` link that will expire in an hour?